src/database.py

```diff
@@ -277,9 +277,9 @@ def setup(service, annex_settings):
     print("- Scrambling root password")
     if password_group is not None:
         root_password = GeneratePassword().get()
-        set_root_password(service, root_password)
     else:
         root_password = 'changeme'
+    set_root_password(service, root_password)
```

Comment:
Do we actually need to have the root user available to us on UAT? What do we use it for atm? Maybe we should just scramble it on UAT also?

Reply:
good question! we could just all of this unconditionally. Any thoughts @MartinEden?

Reply:
Oh I got overexcited and merged this. We could do this as part of VIMC-1726